.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
You should remove this file so far you're not using it, you're using `eslintrc.json` in `stickler.yml`